fvm/src/kernel/default.rs

```diff
@@ -743,11 +779,13 @@ where
 {
     #[allow(unused)]
     fn get_randomness_from_tickets(
-        &self,
+        &mut self,
         personalization: DomainSeparationTag,
         rand_epoch: ChainEpoch,
         entropy: &[u8],
     ) -> Result<[u8; RANDOMNESS_LENGTH]> {
+        self.call_manager
+            .charge_gas(self.call_manager.price_list().on_extern_traversal())?;
```

Comment:
@raulk we need to discuss this as well. We need to be charging a randomness fee (base + multiplier * randomness_size) not an extern traversal fee.

Reply:
Discussing this in sync: We're leaning towards having a flat_fee (that includes the extern cost) + a per_byte_fee.